eos-updater-avahi/eos-updater-avahi.path.in

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Endless OS Avahi Advertisement Updater
+
+[Path]
+# Deployments
+PathChanged=/ostree/repo/refs/heads/ostree/0/0/0
+PathChanged=/ostree/repo/refs/heads/ostree/0/0/1
+PathChanged=/ostree/repo/refs/heads/ostree/0/1/0
+PathChanged=/ostree/repo/refs/heads/ostree/0/1/1
+PathChanged=/ostree/repo/refs/heads/ostree/1/0/0
+PathChanged=/ostree/repo/refs/heads/ostree/1/0/1
+PathChanged=/ostree/repo/refs/heads/ostree/1/1/0
+PathChanged=/ostree/repo/refs/heads/ostree/1/1/1
```

Comment:
What do these paths correspond to? I imagine that they are here to start the service whenever a deployment changes, but perhaps you can include a link or some documentation about why these are those we care about?